src/rpc/RPC.h

```diff
@@ -162,6 +162,32 @@ class AccountNotFoundError : public std::exception
     }
 };
 
+enum warning_code {
+    warnUNKNOWN = -1,
+    warnRPC_CLIO = 2001,
+    warnRPC_OUTDATED = 2002,
+};
+
+struct WarningInfo
+{
+    constexpr WarningInfo() : code(warnUNKNOWN), message("unknown warning")
+    {
+    }
+
+    constexpr WarningInfo(warning_code code_, char const* message_)
+        : code(code_), message(message_)
+    {
+    }
+    warning_code code;
+    Json::StaticString message;
```

Comment:
This can just be a string I think. We don't want to introduce new instances of ripple::Json

Reply:
Fixed